### PR TITLE
Vendor

### PR DESCRIPTION
**Pull request by enxebre**
Revendor to fetch https://github.com/openshift/cluster-api/pull/35 and https://github.com/openshift/cluster-api-actuator-pkg/pull/78

https://github.com/openshift/cluster-api/pull/35 dropped the cluster object from machine.openshift.io as the cluster object is still required programatically via actuator interface this uses the one under cluster.k8s.io

### CODE REVIEW

File: cmd/libvirt-actuator/main.go [hunk 1]
```diff
@@ -25,6 +25,7 @@ import (
 	apiv1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
+	clusterv1 "github.com/openshift/cluster-api/pkg/apis/cluster/v1alpha1"
 	machinev1 "github.com/openshift/cluster-api/pkg/apis/machine/v1beta1"
```
Comment (paulfantom) [inline cmd/libvirt-actuator/main.go]: not used

**Comment by paulfantom** [comment]
Also tests in https://github.com/openshift/cluster-api-provider-libvirt/blob/master/test/machines/machines_test.go need this change

**Comment by ingvagabund** [comment]
```
I0503 10:59:10.703151       1 actuator.go:154] Deleting machine "cluster-8699cdd0-master-machine-8f8c04"
E0503 10:59:10.811067       1 actuator.go:108] Machine error: error creating libvirt client: virError(Code=38, Domain=7, Message='Cannot write data: Broken pipe')
E0503 10:59:10.811172       1 controller.go:207] Failed to delete machine "cluster-8699cdd0-master-machine-8f8c04": error creating libvirt client: virError(Code=38, Domain=7, Message='Cannot write data: Broken pipe')
```

**Pull request merged**